publish-package.sh

```diff
@@ -5,20 +5,20 @@ set -e
 CURRENT_BRANCH=`git name-rev --name-only HEAD`
 
 if [ $CURRENT_BRANCH != 'master' ] ; then
-  echo "Build not on master. Skipped bower-chosen release"
+  echo "Build not on master. Skipped chosen-package release"
   exit 0
 fi
 
 CHOSEN_VERSION=`git tag --sort=v:refname | tail -1`
 
-git config --global user.email "notmyemail@bower-chosen.lol"
-git config --global user.name "bower-chosen"
-
 git clone https://pfiller:${GH_TOKEN}@github.com/harvesthq/bower-chosen.git
 rm -rf bower-chosen/*
-cp public/bower.json public/*.png public/chosen.jquery.js public/chosen.css bower-chosen/
+cp README.md public/*.json public/*.png public/chosen.jquery.js public/chosen.css bower-chosen/
 cd bower-chosen
 
+git config user.email "chosen@harvesthq.com"
```

Comment:
So the old email address was at least obviously a fake one — this one is also fake, but doesn't look like it.

Do we need an email address at all?